keps/sig-contributor-experience/0034-artifact-management.md

```diff
@@ -96,18 +96,17 @@ will serve both binary and container image downloads. For container images, the
 will redirect users to the appropriate geo-located container registry. For binary artifacts, 
 the HTTP redirector will redirect to appropriate geo-located storage buckets.
 
-To facilitate artifact promotion, each project, as necessary, will be given acce
-ss to a
-project staging area relevant to their particular artifacts (either storage buck
-et or image 
+To facilitate artifact promotion, each project, as necessary, will be given access to a
+project staging area relevant to their particular artifacts (either storage bucket or image 
 registry). Each project is free to manage their assets in the staging area however they feel
 it is best to do so. However, end-users are not expected to access artifacts through the
 staging area.
 
 For each artifact, there will be a configuration file checked into this repository. When a
 project wants to promote an image, they will file a PR in this repository to update their
 image promotion configuration to promote an artifact from staging to production. Once this
-PR is approved, automation that is running in the k8s project infrastructure (e.g. https://github.com/GoogleCloudPlatform/k8s-container-image-promoter) will pick up this new
+PR is approved, automation that is running in the k8s project infrastructure (e.g. 
+https://github.com/GoogleCloudPlatform/k8s-container-image-promoter) will pick up this new
 configuration file and copy the relevant bits out to the production serving locations.
 
 Importantly, if a project needs to roll-back or remove an artifact, the same process will
```

Comment:
The Promoter can handle demotions already if you let it delete extra tags (you would just define a manifest that removes the promoted tag, then re-run the promoter against this Promoter manifest) --- and then run garbage collection to delete all un-tagged images from the destination registry. To be fair, garbage-collecting an entire registry seems a bit scary and needlessly sledge-hammery, so we could limit the scope of garbage collection in the future.

Reply:
sounds good.